creator-node/src/fileManager.js

```diff
@@ -105,7 +147,7 @@ async function saveFileToIPFSFromFS (req, srcPath, fileType, sourceFile, transac
  *  - If file is not available via IPFS try other cnode gateways for user's replica set.
  *  - Add file to local ipfs node if not already there.
  */
-async function saveFileForMultihash (req, multihash, expectedStoragePath, gatewaysToTry = []) {
+async function _saveFileForMultihashHelper (req, multihash, expectedStoragePath, gatewaysToTry = []) {
```

Comment:
if this helper isn't called anywhere else, why do we need a new function?

Reply:
we don't. i thought we might but turned out not to be the case. i'll consolidate